### ISSUE TITLE
Can/should we make Jira read-only on migration to GitHub issues?

### ISSUE THREAD

**Issue by mikemccand** [apache/lucene-jira-archive#29]
I think it is crazy trappy to leave Jira as writable after "switching" to GitHub issues.  New users, people seeing old emails in archives and clicking on Jira links, old links in our Wiki/blog posts that we fail/forget to update, will all entice new users to make changes in Jira.  I think this is really quite dangerous, and would specifically impact new users (not us old timers) which is the worst possible impact since our community grows only at its periphery of new users/contributors/developers.

In fact, I don't think we can even call it a switch/migration if we have two writable issue tracking systems.

Second, it is apparently not hard (at least, I heard from @rmuir that people do this by accident sometimes!  I have not researched much myself) to restrict the workflow of Jira so that nothing is writable.

We should also [add a comment to every Jira pointing to the migrated GitHub issue](https://github.com/apache/lucene-jira-archive/issues/15), and perhaps ideally (lower priority) [automatically redirect Jira issues to the right GitHub issue](https://github.com/apache/lucene-jira-archive/issues/28).

**Comment by mikemccand** [comment]
This [Atlassian JIra documentation looks like a great start](https://confluence.atlassian.com/jirakb/how-to-make-jira-issues-read-only-using-workflow-properties-1063568771.html)!  Not sure if we can update these properties ourselves or we need an Infra ticket, but it looks quite simple.

**Comment by mocobeta** [comment]
I just wanted to let you know that I'm not able to edit the Jira configuration such as workflow or issue template (I don't have permission). So anyway, I have to pass it to you after GitHub issue is lifted.

**Comment by mikemccand** [comment]
Hmm OK let me see if I have permissions ;)

**Comment by mikemccand** [comment]
Have we confirmed that INFRA is able to make the whole Jira project read-only after we are done migrating?  Is it just a matter of opening a ticket and it's quick/easy?  From my (limited) online digging I am not so sure it is easy ;)

**Comment by mikemccand** [comment]
Maybe we just open an INFRA issue now, but make it clear not to actually do it yet, to get it on their radar?

**Comment by mocobeta** [comment]
I would open two issues at the same time when we ask infra to start the migration; one for running the import script, and one for making Jira read-only. Anyway, it'll need some time to explain our plan and perform the migration.

**Comment by mocobeta** [comment]
I talked with Infra on the Slack channel.
Jira will be read-only on Monday, August 22 at 8:00 UTC as I sent to the mail lists.

**Comment by mikemccand** [comment]
Wooooot!  Thank you @mocobeta!

**Comment by uschindler** [comment]
How do we now add the final message to all issues in JIRA?

**Comment by mocobeta** [comment]
I still have write access to Lucene Jira. Once Jira notification is disabled (I opened an Infra Jira to modify the notification schema), I will run the script to add comments to each Jira issue.

Noone except for me have write access to Lucene Jira, it is now effectively read-only.
